### PR TITLE
Action object get_config method

### PR DESCRIPTION
**Pull request by Maunty**
Fix method _get_config
method returned wrong values if config contained group value type

### CODE REVIEW

File: adcm_client/objects.py [hunk 1]
```diff
@@ -648,9 +647,14 @@ class Action(BaseAPIObject):
     config = None
 
     def _get_config(self):
-        config = {}
+        config = dict()
```
Comment (a-alferov) [inline adcm_client/objects.py]: Why was {} replaced by dict()

Comment (Maunty) [reply adcm_client/objects.py]: I rewrote this peace several times and just of my personal preferences wrote this way.

Comment (a-alferov) [reply adcm_client/objects.py]: Ok.

Comment (Maunty) [reply adcm_client/objects.py]: Btw. Literal using is faster, but there is a difference only when we compare between several million iterations. If it is one like in our case i don`t think that there are any actual reasons to chose literal using or function call.

Comment (a-alferov) [reply adcm_client/objects.py]: You have to write 4 characters more.

**Pull request merged**